R-package/inst/make-r-def.R

```diff
@@ -71,7 +71,7 @@ invisible(file.remove(OBJDUMP_FILE))
 # see https://www.cs.colorado.edu/~main/cs1300/doc/mingwfaq.html
 start_index <- which(
     grepl(
-        pattern = "[Ordinal/Name Pointer] Table"
+        pattern = "[Ordinal/Name Pointer] Table"  # nolint
```

Comment:
recommend using `# nolint: portable_paths_linter.` for this to help prevent other lints from cropping up on the same line & being hidden :)

Reply:
oooo I didn't know that it was possible to target specific linters like that, [the way you can with flake8](https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors).

Thanks for letting us know! I'll put up a PR doing that.

Reply:
It won't actually do anything yet -- it's part of the 3.0.0 release. But it doesn't hurt to start using it now!

Reply:
Got it, thanks for letting us know about this!